docs/excel/excel-add-ins-ranges-set-get.md

```diff
@@ -47,6 +47,102 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
+## Select the edge of a used range
+
+An add-in can select the edge of the used range, based on the currently selected range. A used range is any cell or series of contiguous cells that have been edited in the worksheet. In the following screenshot, the used range is the table with values in each cell, **C5:F12**. The empty cells outside this table are outside the used range.
+
+![A table with data from C5:F12 in Excel.](../images/excel-ranges-used-range.png)
+
+### Select the cell at the edge of the current used range (online-only)
+
+> [!NOTE]
+> The `Range.getRangeEdge` method is currently only available in ExcelApiOnline 1.1. To learn more, see [Excel JavaScript API online-only requirement set](../reference/requirement-sets/excel-api-online-requirement-set.md).
+
+The following code sample shows how use the [Range.getRangeEdge](/javascript/api/excel/excel.range#getRangeEdge_direction__activeCell_) method to select the cell at the furthest edge of the current used range, in the direction up. This action matches the result of using the Ctrl+Arrow key keyboard shortcut while a range is selected.
+
+```js
+Excel.run(function (context) {
+    // Get the selected range.
+    var range = context.workbook.getSelectedRange();
+
+    // Specify the direction with the `KeyboardDirection` enum.
+    var direction = Excel.KeyboardDirection.up;
+
+    // Get the active cell in the workbook.
+    var activeCell = context.workbook.getActiveCell();
+
+    // Get the top-most cell of the current used range.
+    // This method acts like the Ctrl+Arrow key keyboard shortcut while a range is selected.
+    var rangeEdge = range.getRangeEdge(
+      direction,
+      activeCell
+    );
+    rangeEdge.select();
+
+    await context.sync();
+}).catch(errorHandlerFunction);
+```
+
+> [!NOTE]
+> If the selected range is outside a used range, the `Range.getRangeEdge` method will select the cell at the furthest edge of the worksheet in the chosen direction.
```

Comment:
I think there might be a danger of missing cases here, if we try to explain beyond "it's what the keyboard shortcuts do". In this example, Crtl+Arrow either goes to the furthest edge or the start of the next used range. It also has this behavior when the selected range is at the edge of a used range. Plus, there's a complication because the getUsedRange returns a single range, when in reality their can be multiple "used ranges".

So, the two approaches would be to enumerate the full behavior list or mimic the strings in the reference docs that just pass the explanation burden to the UI.